### PR TITLE
README fixes

### PR DESCRIPTION
**Pull request by fishcharlie**

### CODE REVIEW

**Comment by traversaro** [comment]
Hello @fishcharlie, thanks a lot for the PR! This is outdated, right?

File: README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ Allow support for repositories that use `git lfs` (default `false`).
 ### `ff_only`
 
 Refuse to merge and exit unless the current HEAD is already up to date or the
-merge can be resolved as a fast-forward (default `false`).
+merge can't be resolved as a fast-forward (default `false`).
```
Comment (fishcharlie) [inline README.md:83]: I'm pretty sure this change is still valid. Right? Or am I missing something here?

Comment (traversaro) [reply README.md:83]: It refuse the merge unless the merge can be resolved as fast forward, the current wording seems correct to me or I am missing something?

**Comment by traversaro** [comment]
I think that to avoid confusion we can close this, if there is any fix that is still valid feel free to re-open the issue, thanks!

**Pull request closed without merging**